src/test/scala/htsjdk/samtools/util/StringUtilTest.scala

```diff
@@ -0,0 +1,111 @@
+package htsjdk.samtools.util
```

Comment:
the original has a license...can we leave it on? (I know that many tests do not have license...)